src/ast/irbuilderbpf.cpp

```diff
@@ -790,5 +790,20 @@ void IRBuilderBPF::CreateOverrideReturn(Value *ctx, Value *rc)
   CreateCall(override_func, { ctx, rc }, "override");
 }
 
+Value *IRBuilderBPF::CreatKFuncArg(Value *ctx,
+                                   SizedType &type,
+                                   std::string &name)
+{
+  ctx = CreatePointerCast(ctx, getInt64Ty()->getPointerTo());
+  Value *expr = CreateLoad(getInt64Ty(),
```

Comment:
As work is being done on getting more architectures support; is this always 64 on all architectures?

Reply:
hm, so the pointer could be 32bit and we would add 64bit number,
not sure how would this be handled.. but if it's a problem we have it
in other places, because I stole this code from retval builtin and
saw it on other places ;-)